### ISSUE TITLE
Doesn't compile on latest nightly

### ISSUE THREAD

**Issue by YaLTeR** [SkylerLipthay/interpolate_idents#16]
`rustc 1.17.0-nightly (b1e31766d 2017-03-03)`

```
   Compiling interpolate_idents v0.1.4
error[E0422]: cannot find struct, variant or union type `SequenceRepetition` in module `syntax::tokenstream`
  --> /home/yalter/.cargo/registry/src/github.com-1ecc6299db9ec823/interpolate_idents-0.1.4/src/lib.rs:71:60
   |
71 |                     TokenTree::Sequence(s.clone(), Rc::new(syntax::tokenstream::SequenceRepetition {
   |                                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `syntax::tokenstream`
   |
   = help: possible candidate is found in another module, you can import it into scope:
             `use syntax::ext::tt::quoted::SequenceRepetition;`

error: no associated item named `Sequence` found for type `syntax::tokenstream::TokenTree` in the current scope
  --> /home/yalter/.cargo/registry/src/github.com-1ecc6299db9ec823/interpolate_idents-0.1.4/src/lib.rs:70:18
   |
70 |                 &TokenTree::Sequence(ref s, ref d) => {
   |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: no associated item named `Sequence` found for type `syntax::tokenstream::TokenTree` in the current scope
  --> /home/yalter/.cargo/registry/src/github.com-1ecc6299db9ec823/interpolate_idents-0.1.4/src/lib.rs:71:21
   |
71 |                     TokenTree::Sequence(s.clone(), Rc::new(syntax::tokenstream::SequenceRepetition {
   |                     ^^^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

error: Could not compile `interpolate_idents`.
```

**Comment by YaLTeR** [comment]
Last working nightly: `2017-03-01`

**Comment by SkylerLipthay** [comment]
This has been fixed with #17 and 0.1.5 has been published. Thanks for the report!
